onkar/onkar.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+	<head>
+
+		<title>Onkar | About Me</title>
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+	</head>
+	
+	<body>
+		
+		<div class="prof-box">
+			<img src="images/profile pic color.jpg" class="prof-pic">
```

Comment:
Please don't put spaces in your file names. It can cause some accidentally wonky behavior and is just a pain to type on the command line